providers/oidc.go

```diff
@@ -34,12 +41,29 @@ func NewOIDCProvider(p *ProviderData) *OIDCProvider {
 var _ Provider = (*OIDCProvider)(nil)
 
 // Redeem exchanges the OAuth2 authentication token for an ID token
-func (p *OIDCProvider) Redeem(ctx context.Context, redirectURL, code string) (s *sessions.SessionState, err error) {
+func (p *OIDCProvider) RedeemStandardFlow(ctx context.Context, redirectURL, code string) (s *sessions.SessionState, err error) {
```

Comment:
this function is supposed to be exactly the same as Redeem function, however the CI check complains it has 6 return, limit is 4 return, so add additional getToken function to decouple the original Redeem function.